### PR TITLE
Adding test Python app for PAT Lifecycle Management API

### PR DESCRIPTION
**Pull request by CatherineOsadciw**
Adding a test Python application for using MSAL to get an AAD session token to authenticate and use the upcoming PAT Lifecycle Management API. 

NOTE: this API is not live yet!  It is currently only accessible to Microsoft internal on PPE.

### CODE REVIEW

File: PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/AppCreationScripts/AppCreationScripts.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Registering the sample apps with Microsoft Identity Platform and updating the configuration files using PowerShell scripts
+
+## Overview
+
+### Quick summary
+
+1. On Windows run PowerShell and navigate to the root of the cloned directory
```
Comment (anydot) [inline PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/AppCreationScripts/AppCreationScripts.md:7]: Paths (such as "root of the cloned directory") should be updated now that the sample app isn't in the root

File: PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```
Comment (anydot) [inline PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/CODE_OF_CONDUCT.md:1]: Can you move the code of conduct into the root of this repo? it's missing there so far

File: PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+    MIT License
```
Comment (anydot) [inline PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/LICENSE:1]: There's an existing license file in the repo's root with MIT license

File: PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.1 BLOCK -->
```
Comment (anydot) [inline PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/SECURITY.md:1]: Should be placed in the repo's root

File: PersonalAccessTokenAPIAppSample/.gitignore [hunk 5]
```diff
@@ -0,0 +1,4 @@
+/ms-identity-python-webapp-master/flask_session
```
Comment (anydot) [inline PersonalAccessTokenAPIAppSample/.gitignore:1]: Let's drop the branch name ("master") from the directory path

**Review by anydot — COMMENTED** [review]
Can you also add a link to this new sample from the /README.md

**Comment by CatherineOsadciw** [comment]
TODO: add link to new content docs article to the README

**Review by anydot — APPROVED** [review]

**Pull request merged**